### ISSUE TITLE
Outgoing integrations stopped working in all_direct_messages since 0.49

### ISSUE THREAD

**Issue by danpospisil** [RocketChat/Rocket.Chat#5553]
Your Rocket.Chat version: 0.49

We use Giphy integration (https://rocket.chat/docs/administrator-guides/integrations/giphy) with following Channel settings:
all_public_channels,all_private_groups,all_direct_messages

It works for public channels, private groups but since 0.49 it doesn't work for direct messages.

**Comment by rodrigok** [comment]
@danpospisil Was it working for direct messages before?

Can you show me an example? How replies the message?

**Comment by rodrigok** [comment]
It works for me if I'm talking with the user who was set to reply or for all direct messages if I set `Impersonate User: true` in integration options to post the gif with the username of the user who called.

**Comment by danpospisil** [comment]
I tried to upgrade to 0.49.1 but the issue is still there. 
It was definitely working before - we used to use them with my colleague a lot in our direct messages. I tried to delete the integration and set it up again but no luck. 

I set debug to Info and there is this exception:
```
rocketchat_1  | Integrations ➔ Outgoing WebHook.info Will execute trigger Giphy to https://api.giphy.com/v1/
rocketchat_1  | Integrations ➔ Outgoing WebHook.info Status code for trigger Giphy to https://api.giphy.com/v1/ is 200
rocketchat_1  | Exception in callback of async function: Error: [invalid-channel]
rocketchat_1  |   at retrieveDirectMessageInfo (/app/bundle/programs/server/packages/rocketchat_integrations.js:1516:9)
rocketchat_1  |   at processWebhookMessage (/app/bundle/programs/server/packages/rocketchat_integrations.js:1568:12)
rocketchat_1  |   at sendMessage (/app/bundle/programs/server/packages/rocketchat_integrations.js:1267:22)
rocketchat_1  |   at /app/bundle/programs/server/packages/rocketchat_integrations.js:1319:9
rocketchat_1  |   at runWithEnvironment (packages/meteor.js:1176:24)
```

**Comment by graywolf336** [comment]
@danielbush Question for you. Would the giphy show up in your converstation with someone else or would it show up from `rocket.cat` user?

**Comment by danpospisil** [comment]
@graywolf336 - it would show up as the user who has send the "!gif" message. In the history I see there is a "BOT" label near the username on post with the actual gif image.

**Comment by graywolf336** [comment]
@danpospisil This should be fixed in the release `v0.49.2` which will come out in a few days.

**Comment by engelgabriel** [comment]
We decided to release `0.49.2` today, and leave what is not finished for `0.49.3`in a few days.
